fiftyone/operators/builtin.py

```
@@ -1690,15 +1690,25 @@
         name = ctx.params.get("name", None)
         description = ctx.params.get("description", None)
         color = ctx.params.get("color", None)
+        view = ctx.params.get("view", None)
+
+        curr_view = view is None
+        if curr_view:
+            view = ctx.view
+        else:
+            view = _parse_view(ctx, view)
```

Ruff

Use ternary operator `view = ctx.view if curr_view else _parse_view(ctx, view)` instead of `if`-`else`-block

Replace `if`-`else`-block with `view = ctx.view if curr_view else _parse_view(ctx, view)`

(SIM108)